app/views/admin/users/_member_index.html.erb

```diff
@@ -79,13 +79,7 @@
                   </div>
                 </dl>
                 <div>
-                  <% if user.organization_memberships.any? %>
```

Comment:
There is known repetition between layouts - refactoring into reusable partials is captured as a task in https://github.com/forem/forem/issues/16911. We promise we'll get there very soon.